apps/chat/src/store/conversations/conversations.selectors.ts

```diff
@@ -705,7 +705,7 @@ export const selectIsConversationsEmpty = createSelector(
   [selectSelectedConversations],
   (conversations) => {
     return conversations.some((conv) => {
-      return conv.messages.length === 0;
+      return conv.messages?.length === 0;
```

Comment:
![image](https://github.com/user-attachments/assets/cc95c7c1-ff57-43f5-9717-703b62ad6d64)

```suggestion
      return !conv.messages || conv.messages.length === 0;
```

Reply:
Changed.
Thank you for the suggestion!